eth/test/integration.ts

```diff
@@ -150,14 +160,34 @@ describe("TreasureHuntCreator", () => {
       let [signature, solutionKey] = await getSignature(deployer, testSolution);
       let { r, v, s } = ethers.utils.splitSignature(signature);
 
-      let instance = await deploy([solutionKey], questsRootCid);
+      let instance = await deploy([solutionKey], keys, questsRootCid);
       await instance.connect(deployer).submit(v, r, s);
 
       let result = await instance._chapterToPlayers(0, 0);
       expect(result).equal(deployer.address);
     });
   });
 
+  describe("submitKey", async () => {
+    it("should add a key to the player bitmap", async () => {
+      const instance = await deploy(solutions, keys, questsRootCid);
+      const { r, v, s } = await getKeySignature(MNEMONICS[0], deployer.address);
+      await instance.connect(deployer).submitKey(v, r, s);
+      let result = await instance._playerToKeys(deployer.address);
```

Comment:
there is no assertion here. is it expected?

Reply:
No, it's a mistake, good catch.